keymaps/julia-client.cson

```diff
@@ -8,6 +8,10 @@
   'cmd-enter': 'julia-client:run-block'
   'shift-enter': 'julia-client:run-and-move'
   'cmd-shift-enter': 'julia-client:run-file'
+  'alt-cmd-enter': 'julia-client:run-cell'
+  'alt-shift-enter': 'julia-client:run-cell-and-move'
+  'alt-shift-j': 'julia-client:next-cell'
+  'alt-shift-k': 'julia-client:prev-cell'
```

Comment:
Hi, I've been triyng this PR and I've setup the following keys which has shown to be very intuitive:
```
'alt-enter': 'julia-client:run-cell'
'alt-shift-enter': 'julia-client:run-cell-and-move'
'alt-down': 'julia-client:next-cell'
'alt-up': 'julia-client:prev-cell'
```